packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -137,14 +204,18 @@ export const ChatScreen = () => {
                 contentContainerStyle={styles.flatListContainer}
                 data={chatMessages}
                 keyExtractor={(message) => message.chat_id}
-                renderItem={({ item, index }) => {
-                  return (
-                    <ChatMessageListItem
-                      message={item}
-                      hasTail={hasTail(item, chatMessages[index - 1])}
-                    />
-                  )
-                }}
+                renderItem={({ item, index }) => (
+                  <ChatMessageListItem
```

Comment:
For consistency w/ web, could the separator live in a fragment as a sibling to the `ChatMessageListem` here rather than inside the `ChatMessageListItem` component? or does that not work? (I can also update the web side to match)

Reply:
can do